starport_template/lib/app_config.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:alan/alan.dart';
+import 'package:grpc/grpc.dart';
+
+class AppConfig {
+  AppConfig({
+    this.lcdUrl = 'http://10.0.2.2',
```

Comment:
- `10.0.2.2` - localhost for android emulators
- `lhttp://localhost` - ios simulator

you'll have to stick with only 1 version, I suggest going with `http://localhost` for both LCD and grpc

Reply:
Okay